src/Bicep.Cli.E2eTests/eslint.config.mjs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import notice from "eslint-plugin-notice";
+import tsParser from "@typescript-eslint/parser";
+import globals from "globals";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import js from "@eslint/js";
+import { FlatCompat } from "@eslint/eslintrc";
```

Comment:
Migrating to the flat config format without using `FlatCompat` would be ideal, but I think we can save that for a future task.

Reply:
Yep